krod/core/memory/memory_manager.py

```
@@ -105,4 +110,105 @@ def search_memory(self, user_id: str, query: str, limit: int = 5) -> List[dict]:
+                "query": query,
+                "response": response,
+                "timestamp": datetime.now(timezone.utc).isoformat(),
+                "memory_id": str(uuid.uuid4())
+            }
+            
+            # Add additional metadata if provided
+            if metadata:
+                memory_data.update(metadata)
+                
+            # Store in vector database
+            result = await self.vector_store.add(
```

Pylint (3.3.7)

[error] Instance of 'VectorStore' has no 'add' member

(E1101)